### PR TITLE
bump-formula-pr: add release notes if found

### PR DESCRIPTION
**Pull request by SMillerDev**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [ ] Have you successfully run `brew style` with your changes locally?
- [ ] Have you successfully run `brew typecheck` with your changes locally?
- [ ] Have you successfully run `brew tests` with your changes locally?

-----
This should allow maintainers to quickly spot important information about a release (if it's on GitHub)

### CODE REVIEW

File: Library/Homebrew/dev-cmd/bump-formula-pr.rb [hunk 1]
```diff
@@ -346,6 +346,24 @@ def bump_formula_pr
...
+      rescue GitHub::API::HTTPNotFoundError
+        nil
+      end
+
+      pr_message += <<~EOS if github_release_data.present?
+        <details>
+          <summary>#{github_release_data["prerelease"] ? "pre" : ""}release notes</summary>
+          #{github_release_data["body"]}
+        </details>
+      EOS
```
Comment (MikeMcQuaid) [inline Library/Homebrew/dev-cmd/bump-formula-pr.rb:364]:
```suggestion
      pr_message += if github_release_data.present?
        pre = "pre" if github_release_data["prerelease"].present?
        <<~XML 
          <details>
            <summary>#{pre}release notes</summary>
            #{github_release_data["body"]}
          </details>
        XML
```

Comment (SMillerDev) [reply Library/Homebrew/dev-cmd/bump-formula-pr.rb:364]: Does present? take the Boolean into account?

Comment (MikeMcQuaid) [reply Library/Homebrew/dev-cmd/bump-formula-pr.rb:365]: @SMillerDev Yes, `present?` is truthy for `true`, a number, a non-empty String/Hash/Array, etc.

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by RandomDSdevel** [comment]
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;I could be wrong, but I just thought I'd note thatthis change feels like it's working in roughly the the same space as the old #2935 and #5744, albeit not as a user-facing '`brew`' command, internal or external, and only covering formulas and casks from GitHub with release notes extractable from GitHub Releases pages.

**Comment by chenrui333** [comment]
Like the idea, but we might need to find a way of replacing all the github issue links in the release notes, currently, it messed up the PR/issue references in homebrew-core repo.

**Comment by MikeMcQuaid** [comment]
> Like the idea, but we might need to find a way of replacing all the github issue links in the release notes, currently, it messed up the PR/issue references in homebrew-core repo.

@chenrui333 Can you provide some examples? Thanks!

**Pull request merged**